lib/Yield/ControlFlow/SVG.cpp

```diff
@@ -313,5 +308,61 @@ std::string yield::svg::controlFlow(const yield::Function &InternalFunction,
 
   sugiyama::layout(ControlFlowGraph, Configuration);
 
-  return exportCFG(ControlFlowGraph, InternalFunction, Binary, Configuration);
+  auto ContentHelper = [&](const yield::Graph::Node &Node) {
+    if (Node.Address.isValid())
+      return yield::ptml::controlFlowNode(Node.Address,
+                                          InternalFunction,
+                                          Binary);
+    else
+      return std::string{};
+  };
+  return exportGraph<true>(ControlFlowGraph, Configuration, ContentHelper);
+}
+
+std::string yield::svg::calls(const yield::CrossRelations &Relations,
```

Comment:
Why not `callGraph`?

Reply:
Mostly because `yield::svg::controlFlow`, I tried to be consistent, changing this into a `callGraph` would mean that we should probably change that one as well.

Reply:
I'd go that route. `controlFlow` sounds good, but just `calls` is a bit confusing. It doesn't return "calls" it returns a "callGraph".

Reply:
Done: https://github.com/revng/revng/pull/191/commits/bb86105a0a29128fcf614b11af0c70cd5271c8c3